### PR TITLE
Remove amdgpu installation from quick start

### PR DESCRIPTION
**Pull request by peterjunpark**
This PR removes ROCm installation instructions using the amdgpu installler from the Quick start page

### CODE REVIEW

File: docs/tutorial/quick-start.rst [hunk 1]
```diff
@@ -28,14 +28,13 @@ For more in-depth installation instructions, refer to :ref:`rocm-install-overvie
...
                         :substitutions:
 
+                        sudo apt update
                         sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
-                        sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
+                        # Add the current user to the render and video groups (see prerequisites).
+                        sudo usermod -a -G render,video $LOGNAME
                         wget https://repo.radeon.com/amdgpu-install/|amdgpu_version|/ubuntu/{{ os_release }}/amdgpu-install_|amdgpu_install_version|_all.deb
                         sudo apt install ./amdgpu-install_|amdgpu_install_version|_all.deb
-                        sudo apt update
```
Comment (dbieleck) [inline docs/tutorial/quick-start.rst]: The "sudo apt update" line needs to stay.  Do not remove it.

File: docs/tutorial/quick-start.rst [hunk 2]
```diff
@@ -23,35 +25,16 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
...
-
-                                sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
-                                sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
-                                sudo apt update
-                                wget https://repo.radeon.com/amdgpu-install/|amdgpu_version|/ubuntu/{{ os_release }}/amdgpu-install_|amdgpu_install_version|_all.deb
-                                sudo apt install ./amdgpu-install_|amdgpu_install_version|_all.deb
-                                sudo amdgpu-install --usecase=graphics,rocm
+                        sudo apt update
+                        sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
+                        # Add the current user to the render and video groups (see prerequisites).
```
Comment (yhuiYH) [inline docs/tutorial/quick-start.rst]: I initially had this comment on a separate line.  Feedback was to put it on the same line.  Let's see if others give the same feedback again :)

**Review by dbieleck — CHANGES_REQUESTED** [review]

File: docs/tutorial/quick-start.rst (hunk 2 above)
Comment (dbieleck) [reply docs/tutorial/quick-start.rst]: Do we really need to have the comment at all?

Comment (yhuiYH) [reply docs/tutorial/quick-start.rst]: we added it from Saad's feedback, if you recall.

File: docs/tutorial/quick-start.rst (hunk 1 above)
Comment (peterjunpark) [reply docs/tutorial/quick-start.rst]: @dbieleck We're making `sudo apt update` the first step. Is that fine with you?

```shell
sudo apt update
sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
# Add the current user to the render and video groups (see prerequisites).
sudo usermod -a -G render,video $LOGNAME
wget https://repo.radeon.com/amdgpu-install/6.1.2/ubuntu/jammy/amdgpu-install_6.1.60102-1_all.deb
sudo apt install ./amdgpu-install_6.1.60102-1_all.deb
sudo apt install amdgpu-dkms rocm
```

File: docs/tutorial/quick-start.rst (hunk 2 above)
Comment (dbieleck) [reply docs/tutorial/quick-start.rst]: Right.  I forgot where that came from.  Then maybe we should leave it as a single line.

File: docs/tutorial/quick-start.rst (hunk 1 above)
Comment (dbieleck) [reply docs/tutorial/quick-start.rst]: No.  You need to have the update after the install of the amdgpu-install script, or the instructions will fail:

...
sudo apt install ./amdgpu-install_6.1.60102-1_all.deb
sudo apt update
sudo apt install amdgpu-dkms rocm

The apt update on the top line can remain.

Comment (peterjunpark) [reply docs/tutorial/quick-start.rst]: @dbieleck Ohh ok, thanks. Fixed.

File: docs/tutorial/quick-start.rst [hunk 3]
```diff
@@ -106,45 +67,24 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
...
-                                sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
-                                sudo amdgpu-install --usecase=graphics,rocm
+                        sudo zypper install kernel-default-devel
+                        sudo usermod -a -G render,video $LOGNAME # Add the current user to the render and video groups
+                        sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
+                        sudo zypper refresh
+                        sudo zypper install amdgpu-dkms rocm
                 {% endfor %}
 
+.. important::
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst]: Let's use a Note here - the Google standard is Tips/Note/Warning/Caution

File: docs/tutorial/quick-start.rst [hunk 4]
```diff
@@ -106,45 +67,24 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
...
+
+   Reboot your system for all settings to take effect.
 
-If you encounter install issues, you can refer to the :doc:`troubleshooting <../how-to/native-install/install-faq>` page.
+If you encounter issues with your installation, refer to :doc:`Troubleshooting <../how-to/native-install/install-faq>`.
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst]:
```suggestion
If you have issues with your installation, see :doc:`Troubleshooting <../how-to/native-install/install-faq>`.
```

File: docs/tutorial/quick-start.rst [hunk 5]
```diff
@@ -106,45 +67,24 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
...
+                        sudo zypper install kernel-default-devel
+                        sudo usermod -a -G render,video $LOGNAME # Add the current user to the render and video groups
+                        sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
+                        sudo zypper refresh
+                        sudo zypper install amdgpu-dkms rocm
                 {% endfor %}
 
+.. important::
+
+   Reboot your system for all settings to take effect.
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst]:
```suggestion
To apply all settings, reboot your system.
```

File: docs/tutorial/quick-start.rst [hunk 6]
```diff
@@ -8,9 +8,11 @@
 ROCm quick start install guide for Linux
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst]:
```suggestion
ROCm quick start installation guide for Linux
```

File: docs/tutorial/quick-start.rst [hunk 7]
```diff
@@ -8,9 +8,11 @@
...
 *************************************************************
 
-For a quick summary on installing ROCm on Linux, choose your preferred operating
-system and install method and follow the steps listed in the table. If you want
-more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
+The following are steps to install ROCm on Linux via your distribution's native package manager. To get
+started with ROCm on Linux, select your operating system and version, then run the provided commands in your
+terminal.
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst]:
```suggestion
This topic provides basic installation instructions for ROCm on Linux using your distribution’s native package manager. Review your required installation instructions by selecting your operating system and version, and then run the provided commands in your terminal.
```

**Review by dbieleck — APPROVED** [review]

File: docs/tutorial/quick-start.rst [hunk 8]
```diff
@@ -5,12 +5,12 @@
 .. _rocm-install-quick:
 
 *************************************************************
-ROCm quick start install guide for Linux
+ROCm quick start installation guide for Linux
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst:8]: Change again:
Quick start installation guide

File: docs/tutorial/quick-start.rst [hunk 9]
```diff
@@ -106,45 +65,24 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
...
+                        sudo zypper refresh
+                        sudo zypper install amdgpu-dkms rocm
                 {% endfor %}
 
+.. note::
+
+   To apply all settings, reboot your system.
 
-If you encounter install issues, you can refer to the :doc:`troubleshooting <../how-to/native-install/install-faq>` page.
+If you have issues with your installation, see :doc:`Troubleshooting <../how-to/native-install/install-faq>`.
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst:88]: New paragraph to add at bottom - link to the post install page.

After completing the installation, review the **post-installation instructions**.

File: docs/tutorial/quick-start.rst [hunk 10]
```diff
@@ -5,12 +5,12 @@
...
-ROCm quick start install guide for Linux
+ROCm quick start installation guide for Linux
 *************************************************************
 
-For a quick summary on installing ROCm on Linux, choose your preferred operating
-system and install method and follow the steps listed in the table. If you want
-more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
+This topic provides basic installation instructions for ROCm on Linux using your distribution’s native package manager. Review your required installation instructions by selecting your operating system and version, and then run the provided commands in your terminal.
+
+For more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
```
Comment (lpaoletti) [inline docs/tutorial/quick-start.rst:13]: Need a new paragraph here for Docker - will send over teams once I have it.

File: docs/tutorial/quick-start.rst (hunk 9 above)
Comment (peterjunpark) [reply docs/tutorial/quick-start.rst:88]: Added in https://github.com/ROCm/rocm-install-on-linux/pull/176

File: docs/tutorial/quick-start.rst (hunk 10 above)
Comment (peterjunpark) [reply docs/tutorial/quick-start.rst:13]: To be added in https://github.com/ROCm/rocm-install-on-linux/pull/176

File: docs/tutorial/quick-start.rst (hunk 8 above)
Comment (peterjunpark) [reply docs/tutorial/quick-start.rst:8]: https://github.com/ROCm/rocm-install-on-linux/pull/176

**Pull request merged**